build_test.go

```diff
@@ -332,17 +331,14 @@ func TestBuildFiles(t *testing.T) {
 		t.Run(fixture.name, func(t *testing.T) {
 			t.Parallel()
 			fixture := fixture
-			tmpdir, err := ioutil.TempDir("", "TestBuildFiles-")
-			if err != nil {
-				t.Fatal(err)
-			}
+			tmpdir := t.TempDir()
 			defer os.RemoveAll(tmpdir)
```

Comment:
not needed, since t.TmpDir() removes for us. https://pkg.go.dev/testing#T.TempDir